lib/taskana-core/src/test/java/acceptance/workbasket/query/QueryWorkbasketAccTest.java

```diff
@@ -575,16 +709,21 @@ Stream<DynamicTest> should_SortQueryAsc_When_OrderingByCustomAttribute() {
 
     ThrowingConsumer<WorkbasketCustomField> test =
         customField -> {
-          List<WorkbasketSummary> results =
-              WORKBASKET_SERVICE
-                  .createWorkbasketQuery()
-                  .orderByCustomAttribute(customField, ASCENDING)
-                  .list();
-
-          assertThat(results)
-              .hasSizeGreaterThan(2)
-              .extracting(w -> w.getCustomField(customField))
-              .isSortedAccordingTo(CASE_INSENSITIVE_ORDER);
+          if (customField != WorkbasketCustomField.CUSTOM_1
```

Comment:
Why do you need to split logic here? Could you create Workbaskets with custom fields 5,6,7,8 in the same way as the 1,2,3,4 ?